src/components/review/data-list/page-client.tsx

```diff
@@ -0,0 +1,127 @@
+    ? new Date(createdBefore).toISOString().split('T')[0]
+    : undefined;
+
+  // Fetch monthly summary data with applied filters
+  const { data, isLoading, isFetching, error } = useMonthlySummaryQuery({
+    dateFrom: appliedFilters.dateFrom,
+    dateTo: appliedFilters.dateTo,
+    districts: appliedFilters.districts,
+  });
+
+  const summaries = data?.monthlySummaries ?? [];
+  const availableDistricts = data?.availableDistricts ?? [];
+
+  const handleDateRangeChange = useCallback((newDateRange: DateRangeOption) => {
+    setDateRange(newDateRange);
+  }, []);
+
+  const handleDistrictsChange = useCallback((districts: string[]) => {
+    setSelectedDistricts(districts);
+  }, []);
+
+  const handleApplyFilters = useCallback(() => {
+    setAppliedFilters({
+      dateFrom,
+      dateTo,
+      districts: selectedDistricts.length > 0 ? selectedDistricts : undefined,
+    });
+  }, [dateFrom, dateTo, selectedDistricts]);
+
+  const handleReview = useCallback((district: string, month: string) => {
+    // This will be handled by the Table component
+    console.log('Review clicked for:', district, month);
+  }, []);
+
+  if (error) {
+    return (
+      <div className="container mx-auto px-4 py-6">
+        <Alert variant="destructive">
+          <AlertCircle className="h-4 w-4" />
+          <AlertDescription>
+            Failed to load review data. Please try again.
+          </AlertDescription>
+        </Alert>
+      </div>
+    );
+  }
+
+  return (
+    <div className="container mx-auto space-y-6 px-4 py-6">
+      <div>
+        <h1 className="text-2xl font-bold">Review Data</h1>
+        <p className="text-muted-foreground">
+          Review monthly surveillance data by district and time period.
+        </p>
+      </div>
+
+      {/* Filters Container */}
+      <Card>
+        <CardHeader>
+          <CardTitle className="text-lg">Filters</CardTitle>
+        </CardHeader>
+        <CardContent>
+          <Filters
+            dateRange={dateRange}
+            onDateRangeChange={handleDateRangeChange}
+            districts={availableDistricts.map(d => ({
+              value: d.name.toLowerCase().replace(/\s+/g, '-'),
+              label: d.name,
+            }))}
+            selectedDistricts={selectedDistricts}
+            onDistrictsChange={handleDistrictsChange}
+            onApplyFilters={handleApplyFilters}
+            disabled={isLoading || isFetching}
+          />
+        </CardContent>
+      </Card>
+
+      {/* Review Tasks Container */}
+      <Card>
+        <CardHeader>
+          <CardTitle className="text-lg">
+            Review Tasks ({summaries.length})
+          </CardTitle>
+        </CardHeader>
+        <CardContent>
+          {/* Loading State */}
+          {(isLoading || isFetching) && <LoadingSkeleton />}
+
+          {/* Review Table */}
+          {!isLoading && !isFetching && (
+            <ReviewTable summaries={summaries} onReview={handleReview} />
```

Comment:
[Suggestion]: Maybe change the names for onReview and handleReview to something that more closely describes their functionality?